backend/typescript/types.ts

```diff
@@ -162,6 +162,11 @@ export type CreateCheckInDTO = Omit<CheckInDTO, "id">;
 
 export type UpdateCheckInDTO = Partial<Omit<CheckInDTO, "id">>;
 
+export enum ShiftType {
+  CHECKIN = "checkIn",
+  SCHEDULING = "scheduling",
+}
+
```

Comment:
fyi @carissa-tang we'll need to add this type on the frontend too when you're identifying the type of the schedule for https://github.com/uwblueprint/community-fridge-kw/pull/179